lib/compat/wordpress-6.2/class-wp-theme-json-resolver-6-2.php

```diff
@@ -57,4 +57,59 @@ public static function _clean_cached_data_upon_upgrading( $upgrader, $options )
 		}
 	}
 
+	/**
```

Comment:
This is part of https://github.com/WordPress/gutenberg/pull/45969 To be removed when that PR lands.